python/taichi/lang/_ndarray.py

```diff
@@ -209,7 +209,7 @@ def _fill_by_kernel(self, val):
     @python_scope
     def _pad_key(self, key):
         if key is None:
-            key = ()
+            return (0,)
```

Comment:
Does it work if we just pass a void tuple?

Reply:
Nice catch! let me add that since both numpy and ti.field support it...

Reply:
I mean can we return `()` instead of `(0, )` here now since the backend supports the case when `indices.size() == 0`?

Reply:
oh I see what you meant, makes sense!